src/QmlControls/ParameterEditorDialog.qml

```diff
@@ -138,6 +149,26 @@ QGCViewDialog {
                 }
             }
 
+            Column {
+                Repeater {
+                    id: bitmaskEditor
+                    model: fact.bitmaskStrings
+                    delegate : QGCCheckBox {
+                        text : modelData
+                        checked : fact.value & fact.bitmaskValues[index]
+                        function check_uncheck_all() {
+                            if (text.toLowerCase() == "all") {
```

Comment:
is the strcmp() done only for this param?  I don't think we would could do anything smarter; right thing IMO would be to fix the value in ardupilot so "all" is actually all. However this means we would be changing the API